content/coordination/onboarding/index.en.md

```diff
@@ -7,7 +7,7 @@ weight: 1
 ## Goal
 
 The goal of this activity is to start a conversation with prospective
-organisers, answer any questions and send information on how to get a
+organizers, answer any questions and send information on how to get a
```

Comment:
"organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reply:
Thanks for letting me know.

Reply:
```suggestion
organisers, answer any questions and send information on how to get a
```
I'll reverse the spelling to UK so we can merge these changes an update the Guide